web/core/components/issues/issue-layouts/quick-action-dropdowns/module-issue.tsx

```diff
@@ -180,6 +180,7 @@ export const ModuleIssueQuickActions: React.FC<IQuickActionProps> = observer((pr
         portalElement={portalElement}
         menuItemsClassName="z-[14]"
         maxHeight="lg"
+        useCaptureForOutsideClick
```

Comment:
Are these drop downs used any where else, other than spreadsheet?

Reply:
yes, but `useCaptureForOutsideClick` is optional.